docs/operation/troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_label: Troubleshooting
+---
+
+This section contains information to help you identify and resolve errors you may encounter.
+
+### Known errors and issues
+
+#### lookup failed for scheduledEventID to activityID
+
+See [Nondeterministic troubleshooting steps](#nondeterministic-troubleshooting-steps)
+
+#### unable to find activityID for the scheduledEventID
+
+See [Nondeterministic troubleshooting steps](#nondeterministic-troubleshooting-steps)
+
+#### nondeterministic Workflow error
+
+See [Nondeterministic troubleshooting steps](#nondeterministic-troubleshooting-steps)
+
+#### Potential deadlock
+
+The Temporal Platform requires that a Workflow Function Execution run within 1 second, or it is considered _deadlocked_.
```

Comment:
This is wrong. Please reference the other troubleshooting documentation. A function does not need to run within 1 second. Just part of it before it completes or yields.